driver/modern_bpf/helpers/store/auxmap_store_params.h

```diff
@@ -989,6 +980,50 @@ static __always_inline void auxmap__store_iovec_size_param(struct auxiliary_map
 	auxmap__store_u32_param(auxmap, total_size_to_read);
 }
 
+static __always_inline void auxmap__store_iovec_size_param_32(struct auxiliary_map *auxmap,
```

Comment:
This is identical to `auxmap__store_iovec_size_param_64`, just using `compat_iovec` instead of `iovec`.
I'd use a macro, or at least prepend both with a disclaimer note in case they need to be updated.